docs/book/src/reference/project-config.md

```diff
@@ -6,7 +6,7 @@ The Project Config represents the configuration of a KubeBuilder project. All pr
 
 ## Example
 
-Following an example of an PROJECT file config file which is the result of an project generated with two APIs using the [Deploy Image Plugin][deploy-image-plugin].
+Following an example of an PROJECT file config file which is the result of an project generated with two APIs using the [Deploy Image Plugin](../plugins/deploy-image-plugin-v1-alpha.md).
```

Comment:
We need to use alias.
We let all links/paths at the bottom of the pages see all docs. 

That make easier we keep the things maintained

Reply:
Got it. Updated it in the next commit @camilamacedo86